proto/info.proto

```diff
@@ -43,4 +43,5 @@ message Info {
   repeated Statistic statistics = 1;      // stats that we collect during request processing
   repeated CodedDescription errors = 2;   // errors that occured during request processing
   repeated CodedDescription warnings = 3; // warnings that occured during request processing
+  bool is_service = 4;                    // was this a service request/response rather than a direct call to the library
```

Comment:
this just marks whether the request was being handled by the server or by a direct library call, which is important to protobuf serialization because the former needs to know some of the options to properly handle serialization so they cant be omitted at certain parts of the processing